API/SearchContact.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+	$inData = getRequestInfo();
+	
+	$searchResults = "";
+	$searchCount = 0;
+
+	//connection to database
+	$servername = "localhost";
+    $username = "username";
+    $password = "password";
+    $dbname = "myDB";
+
+    // Create connection
+    $conn = new mysqli($servername, $username, $password, $dbname);
+	if ($conn->connect_error) 
+	{
+		returnWithError( $conn->connect_error );
+	} 
+	else
+	{
+		$contactName = "%" . $inData["search"] . "%";
+		$stmt = $conn->query("SELECT * FROM Contacts WHERE firstName like '$contactName'");
```

Comment:
SQL Convention, keywords should be uppercase.
```suggestion
		$stmt = $conn->query("SELECT * FROM Contacts WHERE firstName LIKE'$contactName'");
```

Reply:
We also should use LIMIT to limit the number of "output" records.